docs/_docs/Starters Academy (LDSSA)/06-Remote-Experience.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Remote Experience
+category: Starters Academy (LDSSA)
+order: 6
+---
+
+
+## Remote, safe, and still a community
+
+As everyone is aware, we are living through the biggest public health crisis in living memory. 
+In the interest of safeguarding the instructors and students physical health, we are doing this 
+batch 100% remote.
+
+In normal times, the course is 90% remote. Of the 10% of the time that we meet up, the bootcamp 
+is the most important. It is where everyone spends a few days together and the community aspect 
+of the LDSSA takes root. We think that this sense of community that you get from meeting people 
+in person is what makes the difference between success and dropout for many students. Most online 
+only courses have very high dropout rates (~90%). Our dropout rate is historically very low (~10%).
+
+So our challenge here is to find a way to maintain a sense of community while staying safe.
+
+## Different personalities, different experiences
+
+Everyone is different. Some people really like socializing. Some people don't like people. 
+Everything in between exists. One interesting thing about this batch is going to be that you 
+have much more control over the type of experience you have. In previous batches, we had in 
+person events that we required attendance to. This created an environment where social was 
+the default. Now it will be the opposite where isolated is the default. That's fine for 
+introverts and not so great for extroverts.
+
+Both experiences will use the same technology. The difference is how you use them. So let's 
+talk first about the tech and then how you can use these tools to craft the type of experience 
+that you want.
+
+## Your experience
+
+There are a few different types of experiences that you can have with this format. A social 
+one or a solitary one. The choice is yours but in the case that you are feeling lost or demotivated, 
+we strongly recommend the social experience. This is how you make the friends that will help you 
+through the course.
+
+### Experience 1: Solitary
+
+This one is easy. If you are overqualified for the course, are super confident that you can 
+breeze through all of the material, and are not a particularly social person then this is the 
+one for you. You leave the auditorium room open, pay attention when presentations are up, and 
+then just work away for the rest of the time.
+
+If you are an introverted person that is not particularly confident in your ability to complete 
+the course, the solitary experience will be very tempting. DON'T DO IT! This is a trap. It's a 
+nice, warm, safe, familiar blanket that will suffocate your chances of completing the course. 
+This is how you end up in your own head with a nice piece of imposter syndrome eating away at 
+your path to success. Bite the bullet and go for Experience 2.
+
+
+### Experience 2: Social
+
+Send direct messages to people. Jump around to all of the specific rooms. If a technical problem 
```

Comment:
A shy person will be scared with this. 😆 
I would smooth it a bit, like saying that we are a welcoming community and that they are free to send direct messages, reach out, etc. instead of "pushing" them to act on it like "send", "jump".

Again, I truly believe the Student Success team should focus on helping out the shy or more insecure students to feel safe, so they can safely start reaching out and not be scared away from the experience. @rjna

Reply:
In the context of the whole page these are advices, and I think they match the tone and provide a good scenario description for people :)

Reply:
Same as above. Redirecting to @rjna in future PR.